pull_requests.rb

```diff
@@ -19,71 +20,71 @@
   opts.on('-v', '--verbose', 'More output') { options[:verbose] = true }
 
   # default filters
-  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') {
+  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') do
     options[:namespace] = 'puppetlabs'
     options[:repo_regex] = '^puppetlabs-'
-  }
+  end
 
-  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') {
+  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') do
     options[:namespace] = 'puppetlabs'
     options[:repo_regex] = OctokitUtils::SUPPORTED_MODULES_REGEX
-  }
+  end
 
-  opts.on('--voxpupuli', 'Select Voxpupuli modules') {
+  opts.on('--voxpupuli', 'Select Voxpupuli modules') do
     options[:namespace] = 'voxpupuli'
     options[:repo_regex] = '^puppet-'
-  }
+  end
```

Comment:
Namespace and repository regex default filters should not be needed anymore with the addition of modules.json